compaction/task_manager_module.cc

```diff
@@ -193,7 +193,8 @@ future<> run_on_table(sstring op, replica::database& db, std::string keyspace, t
     std::exception_ptr ex;
     tasks::tmlogger.debug("Starting {} on {}.{}", op, keyspace, ti.name);
     try {
-        co_await func(db.find_column_family(ti.id));
+        auto tptr = replica::table_ptr(db, ti.id);
```

Comment:
nit, the commit message reads:

> To ensure the table will be dropped while ...

i think it misses a "not".

Reply:
fixed in v2